sdk/face/azure-ai-vision-face/azure/ai/vision/face/__init__.py

```diff
@@ -16,6 +17,7 @@
 from ._patch import patch_sdk as _patch_sdk
 
 __all__ = [
+    "FaceAdministrationClient",
```

Comment:
Since this time you add operation group to client so the folder `_operations` is renamed to `operations`. However, the existing `_operations` will not be deleted automatically, you have to delete it manually. 
nit: it is one-time and next regeneration don't need the manual work anymore.

Reply:
nit: You need to copy content of existing `_patch.py` under `_operations` folder to `_patch.py` under new `operations` folder.

Reply:
Updated